resource-definitions/terraform-driver/runner/s3-ext-runner-refs.tf

```diff
@@ -0,0 +1,72 @@
+resource "humanitec_resource_definition" "aws_terraform_external_runner_resource_s3_bucket" {
```

Comment:
Shouldn't this Resource Definition have a `runner_mode = "custom-kubernetes"` property so it uses a hosted runner?

Reply:
100% thanks for noticing this!

Reply:
Updated